web/app/scripts/filterbar/filterbar.html

```diff
@@ -8,8 +8,17 @@
                 <li date-range-field class="dropdown"></li>
             </ul>
             <ul class="nav navbar-nav" ng-repeat="(key, value) in filterbar.filterables">
-                <li ng-if="value.format=='number'" numeric-range-field label="key" data="value" class="dropdown"></li>
-                <li ng-if="value.fieldType=='selectlist'" options-field label="key" data="value" class="dropdown"></li>
+                <li ng-if="value.format=='number'"
```

Comment:
To align with our js standards, this should be `value.format === 'number'`, same thing with the selectlist (I realize this wasn't added here, but would still be worth updating).